crates/op-rbuilder/src/tests/framework/txs.rs

```diff
@@ -145,16 +153,24 @@ impl TransactionBuilder {
     }
 
     pub async fn send(self) -> eyre::Result<PendingTransactionBuilder<Optimism>> {
+        let with_reverted_hash = self.with_reverted_hash;
         let bundle_opts = self.bundle_opts;
         let provider = self.provider.clone();
         let transaction = self.build().await;
+        let txn_hash = transaction.tx_hash();
         let transaction_encoded = transaction.encoded_2718();
 
         if let Some(bundle_opts) = bundle_opts {
             // Send the transaction as a bundle with the bundle options
             let bundle = Bundle {
                 transactions: vec![transaction_encoded.into()],
+                reverted_hashes: if with_reverted_hash {
+                    vec![txn_hash]
+                } else {
+                    vec![]
+                },
```

Comment:
Should the `BundleOpts` also have a field with reverting txs to be more resembling the real public API and alternatively extract that option from the bundle opts?